_minutes/2025-07-31-wecg.md

```diff
@@ -0,0 +1,172 @@
+ * [devlin] Mostly just trying to say that I'm not convinced increasing the native messaging limit is the right path.
+ * [rob] Are we willing to raise the limit from 1 MB (native app-to-browser) and 64 MB (browser-to-native app).
+ * [devlin] Hesitant to raise beyond 64MB because copying string buffers larger than that is nontrivial and has performance implications.
+ * [rob] 64MB is a reasonable limit from extensions to native messaging host, but the opposite direction is 1MB. Are we in favor of raising that limit to 64MB?
+ * [devlin] Have to look into why we are at 1 MB, but in principle amendable to raising the limit to 64 MB.
+ * [kiara] 64 MB in either direction works for Safari, as long as 80 MB is not exceeded (due to xpc service limits).
+ * **Consensus**: alignment on raising limit to 64 MB in either direction.
+ * [rob] Do we want to set an environment variable or something to expose limits to native applications?
+ * [tomislav] They can always test the limits manually, so there's no reason to hide this variable.
+
+[Issue 855](https://github.com/w3c/webextensions/issues/855): Proposal: Clarify the scale parameter for tabs.captureVisibleTab
+
+ * [devlin] We added “rect” for visible reason, and noticed that “scale” does something different. Our implementation right now () uses “scale” to scale from device independent pixels to pixels, opposed to actually upscale the captured region.
+ * [rob] Tomislav, I think you touched this last, do you know anything about it?
+ * [tomislav] We use it as a scale parameter for dpi calculations.
+ * [devlin] If you have device pixels and css pixels 1:1, and pass a scale of 2, what is the captured region in Firefox?
+ * [tomislav] Should behave as if you manually changed the scale to 200%.
+ * [devlin] In that case, would it change the captured region too?
+ * [rob] That's strange. That would also imply reflow and rerender of the content, whereas this API is supposed to just capture what is displayed.
+ * Action item: Tomislav to to investigate how the current implementation works and follow up on the bug.
+
+[Issue 856](https://github.com/w3c/webextensions/issues/856): Inconsistency: browser.downloads.download API can ignore file name extension if MIME is known type
+
+ * [rob] It seems the request is to be able to set a specific file name. Firefox's file name handling is more permissive than Chrome's. I know Chrome will rename the filename in more cases.
+ * [devlin] I'm surprised that we strip the custom file extension. I don't know if there are extensions reliant on this behavior. What if the MIME type and file extension don't match?
+ * [rob] I wonder if this could be overridden using the onDetermineFilename event in Chrome. Firefox doesn't support this, but Chrome does. If you're concerned about existing extensions, we could introduce a parameter to explicitly change behavior.
```

Comment:
```suggestion
 * [rob] I wonder if this could be overridden using the onDeterminingFilename event in Chrome. Firefox doesn't support this, but Chrome does. If you're concerned about existing extensions, we could introduce a parameter to explicitly change behavior.
```